docs/source/tutorials/token_classification/entity_recognition_training.ipynb

```diff
@@ -0,0 +1,737 @@
+    "        \n",
+    "sentence_tokens = [sentence_tokens[i] for i in range(len(sentences))] \n",
+    "sentence_probs = [sentence_probs[i] for i in range(len(sentences))] "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "754a24f4",
+   "metadata": {},
+   "source": [
+    "Most tokenizers partition sentences into subword-level tokens without altering the characters. However, you should verify whether any characters are modified, particularly for edge cases such as single or double quotations. In this example, double quotations `\"` are broken down into two `'`s or two `` ` ``s. Run the following to verify if any of the characters in the sentences has been modified. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "237ff9c2",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "True\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(modified(given_words, sentence_tokens)) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b498649",
+   "metadata": {},
+   "source": [
+    "Given that some characters have been modified, we need to further map `sentence_tokens` back to the original characters. Note that such mapping is different for different models, and is usually not required by most tokenizers. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "a09fe4c5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# collapse this \n",
+    "replace = [('#', ''), ('``', '\"'), (\"''\", '\"')] \n",
+    "sentence_tokens = [[process_token(token, replace) for token in sentence_tokens[i]] for i in range(len(sentences))] \n",
+    "\n",
+    "for i in range(len(sentences)): \n",
+    "    short = ''.join(given_words[i]) \n",
+    "    if \"''\" in short: \n",
+    "        processed_tokens, processed_probs = [], [] \n",
+    "        for token, prob in zip(sentence_tokens[i], sentence_probs[i]): \n",
+    "            if token != '\"': \n",
+    "                processed_tokens.append(token) \n",
+    "                processed_probs.append(prob) \n",
+    "            else: \n",
+    "                for _ in range(2): \n",
+    "                    processed_tokens.append(\"'\") \n",
+    "                    processed_probs.append(prob) \n",
+    "        sentence_tokens[i] = [token for token in processed_tokens] \n",
+    "        sentence_probs[i] = np.array([prob for prob in processed_probs]) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "00edbe08",
+   "metadata": {},
+   "source": [
+    "In this example, we are more interested in severe types of mislabels, such as `B-LOC` vs. `B-PER`, instead of `B-LOC` vs. `I-LOC`. Therefore, we discard the `B-` and `I-` prefixes, and get the model-predictive probabilities for each subword-level token. The merged entities are `[O, MIS, PER, ORG, LOC]`. As shown below: \n",
```

Comment:
Rephrase last two sentences:

The merged entities are `[O, MIS, PER, ORG, LOC]`, which correspond to the classes in our token classification task.  In a [subsequent notebook](TODO:link), we will use the probabilistic predictions from this trained model to identify instances where the class label was incorrectly chosen for particular tokens.  As shown below: